lnwire/features.go

```diff
@@ -101,6 +101,16 @@ const (
 	// HTLC.
 	MPPOptional FeatureBit = 17
 
+	// AnchorsRequired is a required feature bit that signals that the node
+	// requires channels to be made using commitments having anchor
+	// outputs.
+	AnchorsRequired FeatureBit = 1336
```

Comment:
need to choose actual xzbit

Reply:
same as for TLV types?

Reply:
I don't think it is defined yet in the spec what the range for experimental feature bits is?

Also I don't think picking something `>= 2^16` is what we want in this case, as they are actual bit indices, which would lead to the serialized feature vector being `~ 2^16 / 8 = 8k` bytes long (correct me if I'm wrong). 

Maybe it makes sense to define some a range `>= 128` for experimental features? The would reserve 8 bytes (64 bits => 32 features?) for spec features. Maybe we should reserve even more.

Reply:
This must still be resolved before we can merge this.

Reply:
Rather it should be resolved before we cut a _release_ with this included. Feature bits are easy to change after the fact.